content/en/docs/Installation/linux.md

```diff
@@ -72,7 +187,7 @@ WantedBy=multi-user.target
 User=<user>
 Group=<group>
 Type=simple
-ExecStart=/opt/navidrome/navidrome --configfile "/var/lib/navidrome/navidrome.toml"
+ExecStart=/opt/navidrome/navidrome --configfile "/etc/navidrome/navidrome.toml"
 WorkingDirectory=/var/lib/navidrome
```

Comment:
Potentially also note here, that you may need to update systemd service if backup is somewhere else

Reply:
Going to leave this open for now and link to exact spot where that documentation lives once its in place.